.github/ISSUE_TEMPLATE/maintainerRequest.yml

```diff
@@ -0,0 +1,83 @@
+name: Maintainer Request
+description: Request membership to help maintain this repository.
+title: Maintainer Request
+labels: ['maintainer-request']
+projects: ['repo-management']
+assignees:
+  - kilasuit
+body:
+- type: markdown
+  attributes:
+    value: |
+      ## Thank you for your interest in joining as a maintainer of this project. 
+
+      ### Please complete the following public form to request we review your application to join as a maintainer. 
+- type: dropdown  
+  id: time
+  validations:
+    required: true
+  attributes:
+    label: Can you provide at least 1 hour per week to help maintain this repository?
```

Comment:
This wording was from the PowerShell/PowerShell Working Group Member request form

I kinda want to change this to infer that this should be something like an average of 1hr each  .... (what's a sensible timeframe here | week | Month | Year)